### PR TITLE
feat: rename CowSwap -> CoW Swap

### PR DESCRIPTION
**Pull request by gomesalexandre**
#### Description

This PR renames enum values from `CowSwap` to `CoW Swap`, as per CoW Swap team request.

#### Issue

- relates to https://github.com/shapeshift/web/issues/3449

#### Screenshots

image
image
image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1111** <a href="https://app.graphite.dev/github/pr/shapeshift/lib/1111" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/lib/1111?utm_source=stack-comment).

File: packages/swapper/src/swappers/cow/CowSwapper.test.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('CowSwapper', () => {
 
   describe('name', () => {
     it('returns the correct human readable swapper name', () => {
-      expect(swapper.name).toEqual('CowSwap')
+      expect(swapper.name).toEqual('CoW Swap')
```
Comment (0xApotheosis) [inline packages/swapper/src/swappers/cow/CowSwapper.test.ts:60]: Can we use the `SwapperName` enum here?

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
Looks good, `web` doesn't use any hardcoded strings for this any more so this should be non-breaking.

Just one requested change to get rid of our hardcoded strings.

File: packages/swapper/src/swappers/cow/CowSwapper.test.ts (hunk 1 above)
Comment (gomesalexandre) [reply packages/swapper/src/swappers/cow/CowSwapper.test.ts]: Ab-solutely - ae7e7726f5e1b3814dce80d4837fc022a8d4c22a 🧙🏽

**Review by 0xApotheosis — APPROVED** [review]
💠

**Pull request merged**